projects/examples/src/app/app.component.html

```diff
@@ -14,19 +14,19 @@
                 <section class="nav-group">
                     <label>Components</label>
                     <ul class="nav-list">
-                        <ng-container *ngFor="let entry of sideNavEntries">
-                            <li>
-                                <a class="nav-link" [routerLink]="entry.path" [routerLinkActive]="['active']">
-                                    {{ entry.title }}
-                                </a>
-                            </li>
-                        </ng-container>
+                        <li *ngFor="let entry of sideNavEntries">
+                            <a class="nav-link" [routerLink]="entry.path" [routerLinkActive]="['active']">
+                                {{ entry.title }}
+                            </a>
+                        </li>
                     </ul>
                 </section>
             </section>
         </nav>
 
         <main class="content-area">
+            <ng-template vcdTabset="[]"></ng-template>
+            <div #customTabsEl vcdTabset=""></div>
```

Comment:
What does this HTML code do?

Reply:
Thanks... Removed

Reply:
I don't know if this code was removed in the most recent commit